test/HostActivationTests/GivenThatICareAboutPortableAppActivation.cs

```diff
@@ -74,7 +74,7 @@ public void Muxer_Exec_activation_of_Build_Output_Portable_DLL_with_DepsJson_Loc
             dotnet.Exec("exec", "--runtimeconfig", runtimeConfig, appDll)
                 .CaptureStdErr()
                 .CaptureStdOut()
-                .Execute()
+                .Execute(true)
```

Comment:
yes you are right, This can also be achieved by adding something like 

```C#
class Command
{
...
private bool fExpectedToFail=false;
public Command ExpectToFail()
        {
           fExpectedToFail  = true;
            return this;
        }
...
}
```
and use as 
```C#
CaptureStdOut().
ExpectToFail.
Execute()

```